public/configs/v1/env.json

```diff
@@ -125,7 +127,8 @@
          "withdrawalGateLearnMore": "[HTTP link to withdrawal gate learn more, can be null]",
          "launchIncentive": "[HTTP link to launch incentive host, can be null]",
          "tradingRewardsLearnMore": "[HTTP link to trading rewards learn more, can be null]",
-         "exchangeStats": "[HTTP link to exchange stats, can be null]"
+         "exchangeStats": "[HTTP link to exchange stats, can be null]",
+         "initialMarginFractionLearnMore": "https://help.dydx.trade/en/articles/8587513-default-margin-calculations-on-dydx-chain"
```

Comment:
This value for mainnet config should be a placeholder

Reply:
ah! thanks didn't even notice this